packages/extract/src/webgl/WebGLExtract.js

```diff
@@ -180,9 +174,12 @@ export default class WebGLExtract
 
         if (renderTexture)
         {
-            textureBuffer = renderTexture.baseTexture._glRenderTargets[this.renderer.CONTEXT_UID];
+            // textureBuffer = renderTexture.baseTexture;//._glRenderTargets[this.renderer.CONTEXT_UID];
```

Comment:
Is this line okay to remove?

Reply:
Yeah with this commented out I don't see where `textureBuffer` is assigned anymore?

Reply:
@GoodBoyDigital could you take a look at this. Is the comment-out a mistake?

Reply:
tweaked! good catch chaps